src/output.rs

```diff
@@ -80,6 +80,36 @@ impl Container {
     pub fn contains_conditions(&self) -> bool {
         self.members.iter().any(|m| m.type_.contains("Vec<Condition>"))
     }
+
+    pub fn can_derive_default(&self, containers: &[Container]) -> bool {
+        if self.is_enum {
+            // Need to drop Default from enum as this cannot be derived.
+            // Enum defaults need to either be manually derived
+            // or we can insert enum defaults
+            return false;
+        }
+
+        for m in &self.members {
+            if !m.type_.contains('<')
+                && !m.type_.contains("::")
+                && m.type_ != "String"
+                && m.type_ != "IntOrString"
+                && m.type_ != "NaiveDate"
+                && m.type_ != "DateTime"
+                && m.type_.chars().next().unwrap_or_default().is_uppercase()
```

Comment:
What is the intent behind this check? You're excluding native types with known defaults here, but what about the rest?

Reply:
`::` -> Likely from a 3rd party library. I don't think kopium actually generates types based on that though.

`<` -> Probably Option, Map, Vec, .... Kopium never generates stuff that takes a type parameter, so if that character is present, Default very likely exists.

`m.type_.chars().next().unwrap_or_default().is_uppercase()` -> If the first character is not uppercase, then it's likely a built-in (`bool`, `i32`, ...) and we can also skip the check.